src/components/WebLogView/WebLogView.tsx

```diff
@@ -10,7 +10,6 @@ import { useDeepCompareEffect } from 'react-use'
 interface WebLogViewProps {
   requests: ProxyDataWithMatches[]
   groups?: GroupType[]
-  activeGroup?: string
```

Comment:
The `activeGroup` prop was not used anywhere so I removed it here and any place it was being passed down.